src/common/config.go

```diff
@@ -175,11 +179,14 @@ func LoadDefaults(olPath string) error {
 			Enable_seccomp:      true,
 		},
 		Trace: TraceConfig{
-			Cgroups: false,
-			Memory:  false,
-			Evictor: false,
-			Package: false,
-			Latency: false,
+			Format:        "default",
+			Log_file_dir:  logfileDir,
+			Cgroups:       false,
+			Memory:        false,
+			Evictor:       false,
+			Package:       false,
+			Latency:       false,
+			Cgroups_level: "INFO",
```

Comment:
Can we just have one setting, not both Cgroups and Cgroups_level?  Perhaps Cgroups_level being set to "" can be used to indicate it is disabled.